.github/workflows/pr-builder.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will build the project on pull requests with tests
 # Uses:
 #   OS: ubuntu-latest
-#   JDK: Adopt JDK 8
+#   JDK: Adopt JDK 11 and Adopt JDK 17
```

Comment:
shall we update those readme saying, the component can be compiled with jdk8,11 and 17